docs/src/components/ExampleCode.js

```diff
@@ -61,28 +68,29 @@ export default function ExampleCode({ code, name }: {| code: string, name: strin
             borderStyle="sm"
             display="flex"
             overflow="hidden"
+            maxHeight={!expanded ? CODE_EXAMPLE_HEIGHT : undefined}
             position="relative"
             ref={codeExampleRef}
             rounding={2}
-            {...(expanded ? {} : { maxHeight: CODE_EXAMPLE_HEIGHT, overflow: 'hidden' })}
```

Comment:
This was setting these as attributes, not styles, so the overflow was being ignored